erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -207,7 +212,8 @@ def _get_debit_credit_dict(label):
 	return _dict(
 		opening = _get_debit_credit_dict(_('Opening')),
 		total = _get_debit_credit_dict(_('Total')),
-		closing = _get_debit_credit_dict(_('Closing (Opening + Total)'))
+		closing = _get_debit_credit_dict(_('Closing (Opening + Total)')),
+		total_closing = _get_debit_credit_dict(_('Total Closing (Dr - Cr)'))
```

Comment:
Let's call it "Closing Balance (Dr - Cr)"

Reply:
Fixed!